openapi-to-har.js

```diff
@@ -105,21 +105,50 @@ const getPayload = function (openApi, path, method) {
 
   if (
     openApi.paths[path][method].requestBody &&
-    openApi.paths[path][method].requestBody.content &&
-    openApi.paths[path][method].requestBody.content['application/json'] &&
-    openApi.paths[path][method].requestBody.content['application/json'].schema
+    openApi.paths[path][method].requestBody.content
   ) {
-    const sample = OpenAPISampler.sample(
-      openApi.paths[path][method].requestBody.content['application/json']
-        .schema,
-      { skipReadOnly: true },
-      openApi
-    );
-    return {
-      mimeType: 'application/json',
-      text: JSON.stringify(sample),
-    };
+    if (openApi.paths[path][method].requestBody.content['application/json'] &&
+      openApi.paths[path][method].requestBody.content['application/json'].schema
+    ) {
+      const sample = OpenAPISampler.sample(
+        openApi.paths[path][method].requestBody.content['application/json']
+          .schema,
+        { skipReadOnly: true },
+        openApi
+      );
+      return {
+        mimeType: 'application/json',
+        text: JSON.stringify(sample)
+      };
+    }
+
+    if (openApi.paths[path][method].requestBody.content['application/x-www-form-urlencoded'] &&
+      openApi.paths[path][method].requestBody.content['application/x-www-form-urlencoded'].schema
+    ) {
+      const sample = OpenAPISampler.sample(openApi.paths[path][method].requestBody.content['application/x-www-form-urlencoded']
+          .schema,
+        {skipReadOnly: true},
+        openApi
+      );
+
+      if (sample === undefined) return null;
+
+      const params = [];
+      Object.keys(sample).map(
+        key => params.push({
+          'name': encodeURIComponent(key).replace('%20', '+'),
```

Comment:
`replace('%20', '+')` will only replace the first occurrence of the string `%20`. I propose we use `text.replace(/\%20/g, '+')` instead. Could you make that change and extend your test to make sure that keys / values with more than one whitespace are handled correctly?

Reply:
Since we only replacing one char do you think I can use `replaceAll` instead of the `text.replace` with the wildcard?
I decided to go with the `replaceAll` to keep the syntax of the callback function but it's your call  😃

Reply:
Ignore my last comment 🤦 
I just realized that when you proposed to use `text.replace(/\%20/g, '+')` instead, you were not referring to a new function but actually the same one with a wildcard instead ... Sorry, I'm a JavaScript noob 😅